nativelib/src/main/scala/java/lang/Object.scala

```diff
@@ -14,22 +14,22 @@ class _Object {
     getClass.getName + "@" + Integer.toHexString(hashCode)
 
   def _getClass(): _Class[_] =
-    new _Class(Type.get(this))
+    new _Class(runtime.getType(this))
```

Comment:
Good point, I've just filed an issue for that. https://github.com/scala-native/scala-native/issues/172